client/components/PostPage.jsx

```diff
@@ -0,0 +1,56 @@
+import Navbar from "./Navbar";
+import styles from './PostPage.module.css';
+
+const PostPage = ({ profileImg, profileName, postContent, upVote, downVote, comment, timeStamp }) => {
+
+
+    const data = {
+        profileImg: "https://cdn-icons-png.flaticon.com/512/3899/3899618.png",
+        profileName: "Profile Name",
+        postContent: "Lorem ipsum, dolor sit amet consectetur adipisicing elit. Cumque quia impedit, voluptas distinctio fuga soluta doloremque non tenetur debitis illo ex molestias aliquid, in cum culpa repudiandae blanditiis dignissimos neque? Maiores libero ratione quia porro inventore repudiandae sint culpa similique!",
+        upVote: "Up Vote",
+        downVote: "Down Vote",
+        comment: "Comment",
+        timeStamp: "18:44 11/12/2023",
+    };
+
+    return (
+        <>
+            <Navbar />
+            <div style={{ padding: "100px 0 0 0" }}>
+                <div className={styles.postContainer}>
+                    <div className={styles.buttonContainer}>
+                        <button className={styles.button} onClick={() => history.back()}>
```

Comment:
regarding the history.back()
What if the user entered to this post page via link when he previously was in YouTube ?
so I click the link, then I press back button, will it navigate back to homepage or back to YouTube ?